std-bits/database/src/main/java/org/enso/database/postgres/PostgresConnectionDetailsSPI.java

```diff
@@ -16,7 +16,7 @@ protected String getTypeName() {
 
   @Override
   protected String getCodeForDefaultConstructor() {
-    return "(Postgres)";
+    return "(Postgres 'localhost' 5432)";
```

Comment:
Adding these defaults here somehow makes all the options display. Without them, no options were displayed and one had to use the Code Editor to set options.